src/Sentry/Platforms/Android/AndroidScopeObserver.cs

```diff
@@ -99,4 +99,9 @@ public void SetUser(SentryUser? user)
             _innerObserver?.SetUser(user);
         }
     }
+
+    public void SetTrace(SentryId traceId, SpanId parentSpanId)
+    {
+        // TODO: This requires sentry-java 8.4.0
```

Comment:
Do we need to follow up later with this once this is added to the Java and Apple SDKs?
If so ideally we create a ticket so folks here can add it later

Reply:
https://github.com/getsentry/sentry-dotnet/issues/4074